content/docs/0.8.x/reference/bridge/_index.md

```diff
@@ -13,58 +13,99 @@ The Keptn Bridge provides an easy way to browse all events that are sent within
+        caption="Evaluation Chart"
+        width="50%">}}
 
-* If the `sh.keptn.events.evaluation-done` event has the label `buildId` attached, the Keptn Bridge reads the value of this label and uses it as label for the x-axis in the *Chart*. If the value of the label is an URL, the label will be displayed as a link, so you can easily link back to the Dynatrace Dashboard for example. 
+* To customize the chart, labels can be used. If the `sh.keptn.events.evaluation-done` event has the label `buildId` attached, the Keptn Bridge reads the value
+  of this label and uses it as label for the x-axis in the *Chart*. If the value of the label is an URL, the label will be displayed as a link, 
+  so you can easily link back to the Dynatrace Dashboard for example. 
 
     {{< popup_image
         link="./assets/buildId.png"
         caption="Use `buildId` as label"
         width="50%">}}
+  
+* If you are using relative SLO values that are based on past evaluations *Ignore for comparison* could come in handy. With this you can exclude an evaluation when 
+  determining the relative values. To do so click on the desired evaluation in the heatmap and click the button *Ignore for comparison*. The evaluation is then
+  removed from the heatmap and the evaluation calculation.
 
-### Environment view
-
-* Next to the services view, the Bridge also provides an environment view. 
+    {{< popup_image
+        link="./assets/services-view-invalidate-evaluation.png"
+        caption="Ignore evaluation for comparison"
+        width="50%">}}
+  
+* For your convenience the SLOs that the evaluation was executed against, as well as the event payload, can be viewed.
 
-* This view allows to easily see the current deployed services on each stage and if there are any errors or pending approvals for one stage.
-By clicking on a stage you can see more information about the deployed services on the specific stage on the right side.
+    {{< popup_image
+        link="./assets/services-view-slo.png"
+        caption="Evaluation SLOs"
+        width="50%">}}
 
     {{< popup_image
-    link="./assets/bridge_environment.png"
-    caption="Keptn Bridge environment view">}}
+        link="./assets/services-view-payload.png"
+        caption="Event Payload"
+        width="50%">}}
   
 ### Evaluation board
 
-* Next to the evaluation comparison as a heatmap or chart in the *Services* view, we provide a link to the *Evaluation Board*, which displays only the evaluation comparison on this stage. 
+* Next to the evaluation comparison as a heatmap or chart in the *Services* view, we provide a link to the *Evaluation Board*, which displays only 
+  the evaluation comparison on this stage. 
   
     {{< popup_image
     link="./assets/bridge_evaluationboard.png"
     caption="Keptn Bridge Evaluation Board">}}
+  
+### Sequences view
+
+* The sequences for the whole Keptn context are shown in this screen. It has a lot of filter possibilities, that can either be selected
```

Comment:
Please change to: `The sequences for the selected project are shown on this screen.`